hollow/src/main/java/com/netflix/hollow/api/codegen/api/HollowDataAccessorGenerator.java

```diff
@@ -68,7 +68,7 @@ public String generate() {
         builder.append("@SuppressWarnings(\"all\")\n");
         builder.append("public class " + className + " extends " + AbstractHollowDataAccessor.class.getSimpleName() + "<" + type  +"> {\n\n");
 
-        builder.append("    public static final String TYPE = \"" + type + "\";\n");
+        builder.append("    public static final String TYPE = \"" + schema.getName() + "\";\n");
```

Comment:
Not very familiar with this code path. Can you plz share example generated DataAccess BEFORE and AFTER?

Reply:
This is to address issue when HollowAPIGenerator.Builder().withClassPostfix("Hollow").

Usage of type of correct on other places but TYPE here is supposed to be matching schema type name.   We can consider renaming TYPE -> SCHEMA_TYPE if that help.   

Ping me and I can provide actual example on when existing code has issue.

Reply:
ACK, thanks